<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddReceiptWindow</class>
 <widget class="QDialog" name="AddReceiptWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add new receipt...</string>
  </property>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadReceiptButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>upload receipt file</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="receiptNameLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Receipt name 
(unique)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
